src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkNotFoundException.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.appengine.cloudsdk;
+
+import com.google.cloud.tools.appengine.api.AppEngineException;
+
+/**
+ * User needs to use a valid Cloud SDK installation.
+ */
+public class CloudSdkNotFoundException extends AppEngineException {
```

Comment:
I could wait on making the old AppEngineException checked, but this one should be checked form the get go since the point of this PR is that clients should be catching and handling this exception. Otherwise it's really not useful.

The alternative would be to rethink throwing the exception and have it return a boolean instead.